pkg/volume/plugins.go

```diff
@@ -151,6 +157,7 @@ type AttachableVolumePlugin interface {
 	VolumePlugin
 	NewAttacher() (Attacher, error)
 	NewDetacher() (Detacher, error)
+	GetDeviceMountRefs(deviceMountPath string) ([]string, error)
```

Comment:
Even if you need this in operation executor, it does not need be added to the volume plugin interface. All instances of this for each plugin will be identical. You can call `volumePlugin.NewMounter` etc from operation executor